quadkey/bq/test/quadkey_integration.js

```diff
@@ -0,0 +1,500 @@
+const assert = require('assert').strict;
+const {BigQuery} = require('@google-cloud/bigquery');
+
+const BQ_PROJECTID = process.env.BQ_PROJECTID;
+const BQ_DATASET_QUADKEY = process.env.BQ_DATASET_QUADKEY;
+
+describe('QUADKEY integration tests', () => {
```

Comment:
Personally I prefer to break down the integration tests in multiple files, it's clearer to see what's being tested in each of them and they are executed in parallel (so everything is much faster). In any case, this looks good.

Reply:
I will split it then as it should be a fast change.